src/embed/Profiteur/Data.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Profiteur.Data where
```

Comment:
This module should be called `Profiteur.DataFile` and it should re-export `Profiteur.DataFile.Internal`.  The same goes for the other `Profiteur.DataFile` module.